README.md

```diff
@@ -6,7 +6,7 @@ The Datadog Chef recipes are used to deploy Datadog's components and configurati
 * Datadog Agent v6.x
 * Datadog Agent v5.x
 
-**Note**: This page may refer to features that are not available for your selected version. Check the README of the
+**Note**: This page may reference features that are not available for your selected version. Check the README of the
```

Comment:
Not sure this is better than "refer to." How about "mention" or "discuss" instead.

Reply:
```suggestion
**Note**: This page may discuss features that are not available for your selected version. Check the README of the
```